mne/bem.py

```diff
@@ -332,3 +337,108 @@ def cost_fun(x, points):
     origin = x_opt[:3]
     radius = x_opt[3]
     return radius, origin
+
+
+# ############################################################################
+# Create BEM surfaces
+
+@verbose
+def make_watershed_bem(subject=None, subjects_dir=None, overwrite=False,
```

Comment:
I don't think we should allow `subject=None` here. It should be explicitly used. (Using the env var is not safe, most mne-pythoners don't use subject env vars as they aren't used anywhere else.)